core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -593,4 +594,55 @@ class SocketServerTest extends JUnitSuite {
     }
   }
 
+  @Test
+  def channelException() {
```

Comment:
Maybe we can make this more descriptive?

Reply:
Removed this test, have a new set of tests for various scenarios.